torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1448,7 +1448,7 @@ void TensorExprKernel::bindConstant(const torch::jit::Value* v) {
       ToDtype(scalar_type));
 
   if (!const_tensor.is_contiguous()) {
-    const_tensor = const_tensor.clone().contiguous();
+    const_tensor = const_tensor.contiguous();
```

Comment:
If it will always clone here if it's not contiguous, but you could have just called clone with contiguous_format for the same effect.

Reply:
Fixed